components/21-atoms/table/table.bindings.js

```diff
@@ -0,0 +1,22 @@
+/** global ResponsiveTable, MobileTable */
+
+'use strict';
+
+(function () {
+  var tablesNodeList = document.querySelectorAll('.responsive-table');
+
+  // Optimise all tables with a wrapper div.responsive-table
+  for (var i = 0; i < tablesNodeList.length; i++) {
+    var table = tablesNodeList[i];
+
+    // Adds accessibility support.
+
+    new ResponsiveTable(table, { // eslint-disable-line no-undef
+      scrollableText: '(scroll to see more)'
+    });
+
+    // Adds a mobile solution using definition lists.
+    new MobileTable(table, {}); // eslint-disable-line no-undef
```

Comment:
* No need to pass in the empty object
* the eslint-disable isn't necessary anymore since you declared it as a global variable at the top of this file

```suggestion
    new MobileTable(table);
```